OpenShift/add-machine-ips.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -x
+set -e
+
+export CLUSTER_NAME=$(hostname -f | cut -d'.' -f 2)
```

Comment:
@sreichar is this a _hard requirement_ to have a provisionhost's fqdn with cluster parts in?

Reply:
That has been my understanding.